content/blog/generative-vs-extractive-models/index.md

```diff
@@ -37,11 +37,11 @@ Closed-source language models like those by OpenAI are only accessible through a
 
 In terms of architecture, broadly speaking, extractive models like those [based on BERT](https://haystack.deepset.ai/blog/the-definitive-guide-to-bertmodels) belong to the same family of language models as their generative counterparts, since both use the Transformers. However, the main distinction lies in the usage of these models. Unlike generative models, extractive models require both a query and a specific piece of text as the foundation for their response. It then encodes the query as a high-dimensional, semantically informed vector exactly like a generative model does. But rather than generating an answer from scratch based on the given context as generative models do, the extractive model marks the section in the text document that, according to the model’s prediction, provides the best answer to the query.
 
-![](hamlet.png)
+![A high-level explanation of how extractive models retrieve answers from a given document. The question 'Who is the main female character in 'Hamlet'?' is asked, and the Reader component, using an extractive language model, answers with 'Ophelia, Polonius's daughter' from the given document.](hamlet.png)
```

Comment:
You don't need to repeat the text on the img. So we could replace the second sentence with something like: The Reader component using an extractive language model, searches for the exact answer in the document and displays it.

Reply:
I believe we need to be as descriptive as possible so that people will have enough information to visualize the image from the text as much as possible.